test/system/030-run.bats

```diff
@@ -1197,6 +1198,12 @@ search               | $IMAGE           |
         run_podman 125 $command --authfile=$bogus $args
         assert "$output" = "Error: checking authfile: stat $bogus: no such file or directory" \
            "$command --authfile=nonexistent-path"
+
+        if [[ "$command" != "logout" ]]; then
+           REGISTRY_AUTH_FILE=$bogus run_podman ? $command $args
+           assert "$output" !~ "checking authfile" \
+              "$command --authfile=nonexistent-path"
```

Comment:
Test description is still wrong (see part 2 of my original request)  but I can clean it up in a future iteration of #19384